docs/external_repos.md

```diff
@@ -0,0 +1,56 @@
+# External Repos
+You may want to include code from an external repository that is outside of your 
+current working directory project. This is supported through the `external_github_repo`
+field in the configuration file. After launch, the repository will be pulled into the 
+evaluations tmp folder and placed on the SYS PATH so that modules can be easily 
```

Comment:
PYTHONPATH or SYSTEM PATH?

Reply:
So it's technically SYS PATH: 
https://docs.python.org/3/library/sys.html#sys.path

It's initialized from PYTHONPATH though

Reply:
I just wanted to make it clear that it was the sys.path from python, and not the docker container's system path.